public/app/features/alerting/unified/api/templateApi.ts

```diff
@@ -30,7 +30,18 @@ export interface AlertField {
   labels: KeyValueField[];
 }
 
-export const templatesApi = alertingApi.injectEndpoints({
+generatedTemplatesApi.enhanceEndpoints({
```

Comment:
I've noticed the new UID is part of the `PUT` response, is there any way we can use that to determine if we should invalidate the previous UID?

Reply:
EDIT: after walking through the generated client I don't think there's much we can do for now other than disabling the alerts